epoch/api/channels_api_usage.md

```diff
@@ -518,3 +521,268 @@ details:
 }
 ```
 
+#### Total balance update events
+
+After the channel has been opened it has a total balance of tokens committed to
+it. This balance is persisted as part of the on-chain channel state. Upon
+closing a channel on-chain, the closing balances of the participants are
+checked against this balance. Under no circumstances can the sum of the closing balances
+be greater than the total balance on-chain.
+
+Participants are able to modify the total balance: the following two functionalities are available:
+
+* deposit - when a participant wants to commit more tokens from his on-chain
+  balance to the channel total balance
+* withdrawal - when a participant wants to transfer tokens out of the channel
+  on-chain balace to one's personal account
+
+### Deposit events
+
+After the channel had been opened any of the participants can initiate a
+deposit. The process closely resembles the [update](#update). The most notable
+difference is the transaction has been co-signed: it is `channel_deposit_tx` and
+after the procedure is finished, it is posted on-chain.
+
+Since both the initiator and responder can deposit tokens, in the scope of this description we
+will call the participant that commits tokens to the channel a depositor and
+the other party - acknowledger. Note that any public key outside of the channel participants
+is considered invalid for the depositor role.
+
+#### Deposit transaction
+The `channel_deposit_tx` is a change to be applied on top of the latest channel state. It also is
+posted on-chain and is included in a block. It has the following structure:
+
+  | Name | Type | Description |
+  | ---- | ---- | ----------- |
+  | channel id | string | ID of the channel|
+  | from | string | depositor's public key |
+  | amount | integer | the amount committed to the channel |
+  | ttl | integer | minimum block height to include the transaction |
+  | fee | integer | fee to be paid to the miner |
+  | state_hash | string | the root of the internal channel state hash after the deposit |
+  | round | integer | the next channel round |
+  | nonce | integer | depositor's nonce |
+
+#### Start deposit
+
+Any of the participants can initiate a deposit. Only requirements are:
+* Channel is already opened
+* No off-chain update/deposit/withdrawal is currently being performed
+* Channel is not being closed or in a solo closing state
```

Comment:
* The deposit must be equal to or greater than zero, and cannot exceed the available balance on the depositor's account.